contracts/axelar-interchain-token-service/src/error.rs

```diff
@@ -0,0 +1,12 @@
+use soroban_sdk::contracterror;
+
+#[contracterror]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
+#[repr(u32)]
+pub enum InterchainTokenServiceError {
```

Comment:
```suggestion
pub enum ContractError {
```
since we aren't using the interface crate anymore, we can give this a common name